database/database.go

```diff
@@ -931,7 +931,7 @@ func (db *Database) createConfigTable() (err error) {
 			fotaUpdateState,
 			sotaUpdateState,
 			desiredInstances) values(?, ?, ?, ?, ?)`,
-		"", "", json.RawMessage{}, json.RawMessage{}, json.RawMessage{}); err != nil {
+		"", "", json.RawMessage{}, json.RawMessage{}, json.RawMessage("[]")); err != nil {
```

Comment:
What is wrong with empty json?

Reply:
we expect array as a result error message